examples/rust/examples/maker/maker.rs

```diff
@@ -168,9 +189,11 @@ async fn run<P: JsonRpcClient + 'static>(
                 rx_quote_response,
             ))
             .await
-            .unwrap()
+            .ok()?
             .into_inner();
 
+        info!("Ready for RFQs from Takers");
+
         // Now we have received the RFQ request stream - loop until it ends.
         while let Ok(Some(quote)) = quote_stream.message().await {
             // Check the chain-id is valid
```

Comment:
the server already does this now, but extra safe to keep here

Reply:
same for ulid check done in validate_rfq, but makers should be doing these checks as well, so keep here

Reply:
Never assume any input to an application you are writing has been checked/is correct. You should always sanitize any input data you program receives before dealing with it.